### DISCUSSION TITLE
diary/school-fourth/

### DISCUSSION THREAD

**Discussion by giscus** [jiyongYoon/jiyongYoon.github.io#9] [Comments]
# diary/school-fourth/

백엔드 부트캠프-제로베이스 백엔드 스쿨 - 5개월차

https://jiyongyoon.github.io/diary/school-fourth/

**Comment by goraneee** [comment]
우수 수강생 추카추카~

**Reply by jiyongYoon** [reply]
감사합니다 고라니님!!! :) 같이 파이팅해봅시다!!!
